src/main/java/seedu/allonus/modules/StudyManager.java

```diff
@@ -168,6 +243,30 @@ public Module addModuleParser(String userInput) {
         }
     }
 
+    public void findModule(String userInput) {
+        String moduleKeyword = userInput.replace("find ","");
+        ArrayList<Module> matches = new ArrayList<>();
+        for (Module m: modulesList) {
+            if (m.toString().contains(moduleKeyword)) {
```

Comment:
If I'm not wrong, the user would be able to enter in a blank keyword to look for and the method may return every item in the list. Perhaps consider edge cases of inappropriate keywords for subsequent commits!